crates/holochain/src/conductor/conductor.rs

```diff
@@ -721,10 +721,10 @@ mod dna_impls {
 
         /// Restart every paused app
         #[cfg_attr(feature = "instrument", tracing::instrument(skip_all))]
-        pub(crate) async fn start_paused_apps(&self) -> ConductorResult<AppStatusFx> {
+        pub(crate) async fn start_apps(&self) -> ConductorResult<AppStatusFx> {
```

Comment:
If this function is starting stopped apps I would prefer it would be called `start_stopped_apps()`, otherwise it sounds like it is starting all apps. Or is this what it's doing?